app/views/admin/users/_list_user.html.erb

```diff
@@ -0,0 +1,8 @@
+<tr>
+  <td scope="row"><%= user.full_name %></td>
+  <td scope="row"><%= user.email %></td>
+  <td scope="row"><%= user.activated %></td>
+  <td scope="row"><%= user.activated_at.to_formatted_s(:db) %></td>
```

Comment:
`https://guides.rubyonrails.org/v2.3/i18n.html`


en:
  date:
    formats:
      default: "%Y-%m-%d"
      short: "%b %d"
      long: "%B %d, %Y"
So, all of the following equivalent lookups will return the :short date format "%B %d":


I18n.t 'date.formats.short'
I18n.t 'formats.short', :scope => :date
I18n.t :short, :scope => 'date.formats'
I18n.t :short, :scope => [:date, :formats]